javascript/javascript1/week1/index.js

```diff
@@ -0,0 +1,77 @@
+// Calc athe human age
+
+const birthYear = document.getElementById("birthYear");
+const selectedYear = document.getElementById("selectedYear");
+const showYear = document.getElementById("showYear");
+const calcAgeButton = document.getElementById("calcAgeButton");
+
+function calcAge() {
+  const result = selectedYear.value - birthYear.value;
+  showYear.classList.add("show");
+  showYear.innerHTML = "Your Age will Be : " + result;
+  return result;
+}
+
+// Calc the dog age
+
+const dogYearOfBirth = document.getElementById("dogYearOfBrith");
+const dogYearFuture = document.getElementById("dogYearFuture");
+const dogYears = document.getElementById("dogYears");
+const humanYears = document.getElementById("humanYears");
+const calcHumanDogAge = document.getElementById("calcHumanDogAgeButton");
+const showAge = document.getElementById("showAge");
+
+function calcDogHumanAge() {
+  const age = dogYears.checked
+    ? (dogYearFuture.value - dogYearOfBirth.value) * 7 + " Dogs Year"
+    : dogYearFuture.value - dogYearOfBirth.value + " Human Years";
+  showAge.innerHTML = "Your dog's old is : " + age;
+  showAge.classList.add("show");
+  return age;
+}
+
+// the house price
+
+const width = document.getElementById("width");
+const depth = document.getElementById("depth");
+const height = document.getElementById("height");
+const gardenSize = document.getElementById("gardenSize");
+const priceHouseButton = document.getElementById("priceHouseButton");
+
+const showRightPrice = document.getElementById("showRightPrice");
+
+priceHouseButton.addEventListener("click", rightPriceForHouse);
+
+function rightPriceForHouse() {
+  // the price for the house without garden:
+  const volumeInMeters =
+    width.value * depth.value * height.value * 2.5 * 1000 +
+    gardenSize.value * 300;
+  showRightPrice.innerHTML =
+    "The right price for the house is : " + volumeInMeters;
+
+  showRightPrice.classList.add("show");
+  return volumeInMeters;
+}
+
+// Ez Namey (Startup name generator) Optional
+
+let firstNamesArray = ["honey", "love", "sweet", "Yammmmiy", "lolo"];
+let secondNamesArray = ["Gary", "ismailo", "Nana", "inyea", "soso"];
+
+const randomNumber = Math.floor(Math.random() * 10) + 0;
```

Comment:
I know the task is technically optional, but i would rather you spend time on an extra exercise, than all the html and css. (At least in this module).